src/control/createIndex.ts

```diff
@@ -0,0 +1,46 @@
+import { IndexOperationsApi } from '../pinecone-generated-ts-fetch';
+import type { ResponseError } from '../pinecone-generated-ts-fetch';
+import { mapHttpStatusError } from '../errors';
+import { builOptionConfigValidator } from '../validator';
+
+import { Static, Type } from '@sinclair/typebox';
+
+const nonemptyString = Type.String({ minLength: 1 });
+const positiveInteger = Type.Integer({ minimum: 1 });
+
+const CreateIndexOptionsSchema = Type.Object({
```

Comment:
I think you mentioned but I forgot, why did we use it instead of TS? to be able to do it runtime when it's already DIST'd?